systemvm/patches/debian/config/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -96,7 +96,7 @@ def _redundant_on(self):
                 d = s.replace(".templ", "")
             CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, s), "%s/%s" % (self.CS_ROUTER_DIR, d))
         CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, "keepalived.conf.templ"), self.KEEPALIVED_CONF)
-        CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, "conntrackd.conf.templ"), self.CONNTRACKD_CONF)
+        CsHelper.copy("%s/%s" % (self.CS_TEMPLATES_DIR, "conntrackd.conf.templ"), self.CONNTRACKD_CONF)
```

Comment:
Hi Daan,

The intent of copy_if_needed is to prevent overwriting the config file if it is preset already. In case of conntrackd, the config file is already present and so it dose not copy the default cloudstack conntrackd template and as a result conntrackd fails to start. For this(copy_if_needed)  to work we should make sure that this file is not present when this code runs for the first time. There may be similar errors related to rvr. I was trying to see if i can address them as a part of larger fix.